pkg/kubectl/cmd/util/printing.go

```diff
@@ -131,7 +131,7 @@ func PrinterForCommand(cmd *cobra.Command, outputOpts *printers.OutputOptions, m
 	// we try to convert to HumanReadablePrinter, if return ok, it must be no generic
 	// we execute AddHandlers() here before maybeWrapSortingPrinter so that we don't
 	// need to convert to delegatePrinter again then invoke AddHandlers()
-	if humanReadablePrinter, ok := printer.(*printers.HumanReadablePrinter); ok {
+	if humanReadablePrinter, ok := printer.(printers.PrintHandler); ok {
```

Comment:
entangling adding handlers and wrapping in this single function doesn't allow adding additional handlers in a calling factory. @smarterclayton, thoughts on the sorting wrapper being added this way?

Reply:
@liggitt will leave this line as-is for now. There is a [proposal for discussing how to address adding handlers to a printer](https://docs.google.com/a/redhat.com/document/d/1SBhkK34UNObC3KLva9aaolonoFGRzqZ6YSeZMU8AZQc/edit?usp=sharing), however a goal would be to eliminate printer handlers altogether [in favor of server-side printers](https://github.com/kubernetes/kubernetes/pull/55637).

ping @smarterclayton